build/assets/drawable/vtmn_flag_au.xml

```diff
@@ -63,5 +38,16 @@
             android:fillColor="#FF000000"
```

Comment:
Seems a lot of white paths are generated black, for several flags like AG, AI, AN, AU and many others

Reply:
Good catch !

Reply:
<img width="890" alt="CleanShot 2023-04-05 at 09 50 27@2x" src="https://user-images.githubusercontent.com/61324309/230016279-fa010caf-e932-4a04-9966-5f8f09d14b0e.png">
On my side I didn't see problems there is white shapes